### PR TITLE
:pray:

### PR DESCRIPTION
**Pull request by Haargeeel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,581 +0,0 @@
...
-
-  const b = 3
-
-  // GOOD
-  const a = 5
-
-  const b = 4
-  ```
-
-  - There should be a space after open curly brace and before close curly brace for objects and destructor[object-curly-spacing]
```
Comment (Haargeeel) [inline README.md:252]: I can't do it. It's too hard and also it wants me to write:
`this.state = { reasonList: []}` whhyyyyy?

Comment (goferito) [reply README.md:252]: I still defend to drop it :freedom:!!! :free:

Comment (goferito) [reply README.md:252]: @mateeyow ?

Comment (mateeyow) [reply README.md:252]: @goferito This is already removed. :)

**Pull request closed without merging**